src/Microsoft.ML.CpuMath/Microsoft.ML.CpuMath.csproj

```diff
@@ -10,6 +10,11 @@
     <LangVersion>7.3</LangVersion>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(TargetFramework)|$(Platform)'=='Release-Intrinsics|netstandard2.0|AnyCPU'">
```

Comment:
What is this change for?

Reply:
I also want to know!  I think one day Visual Studio added this to the .csproj file on its own...  I guess it tries to optimize stuff?  But not sure...

Reply:
Should I delete this?

Reply:
Yes, please.